packages/firebase_remote_config/example/lib/main.dart

```diff
@@ -0,0 +1,61 @@
+import 'dart:async';
+
+import 'package:firebase_remote_config/firebase_remote_config.dart';
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(new MaterialApp(
+      title: 'Remote Config Example',
+      home: new FutureBuilder<RemoteConfig>(
+        future: setupRemoteConfig(),
+        builder: (BuildContext context, AsyncSnapshot<RemoteConfig> snapshot) {
+          return snapshot.hasData
+              ? new WelcomeWidget(remoteConfig: snapshot.data)
+              : new Container();
+        },
+      )));
+}
+
+class WelcomeWidget extends AnimatedWidget {
+  final RemoteConfig remoteConfig;
+
+  WelcomeWidget({this.remoteConfig}) : super(listenable: remoteConfig);
```

Comment:
Flutter style is to [put constructors first in the class](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#constructors-come-first-in-a-class), before the members.

Reply:
Done!